### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by anselmbradford**
## Removals

- Removed unused inline brand styles.
- Removed unused inline JS.

## Changes

- Moves inline JS to modules.

## Testing

- `gulp clean && gulp build`
- Should see no JS errors in the console for the steps (expect the confirmation page, which has an existing error for cfpbAriaAttribute) and it should work as expected.

## Review

- @niqjohnson

### CODE REVIEW

File: src/static/js/main.js [hunk 1]
```diff
@@ -1,198 +1,39 @@
...
-    _showCreateTab();
-  } );
-}
+var globalModals = require( './modules/global-modals' );
+var stepWelcome = require( './modules/step-welcome' );
+var stepSelectProduct = require( './modules/step-select-product' );
+var stepSelectIssue = require( './modules/step-select-issue' );
+var stepWhatHappened = require( './modules/step-what-happened' );
+var stepCompanyInformation = require( './modules/step-company-information' );
+var stepYourInformation = require( './modules/step-company-information' );
```
Comment (niqjohnson) [inline src/static/js/main.js]: Should this be:

```js
var stepYourInformation = require( './modules/step-your-information' );
```

Also, it looks like that module has a "2" at the end of its file name right now: `step-your-information 2.js`

**Comment by niqjohnson** [comment]
Love all the reorganization and cleanup! This inspired me to vacuum my office while I was in the middle of reviewing.

One line note above about `step-your-information`. Otherwise, I think everything is working as before.

File: src/static/js/main.js (hunk 1 above)
Comment (anselmbradford) [inline src/static/js/main.js]: Ugh, yup.. looks like something didn't save.

**Comment by anselmbradford** [comment]
Haha, great! I'll fix that and merge

**Pull request merged**